src/EditorFeatures/Core/Implementation/NavigateTo/NavigateToItemProvider.NavigateToItemDisplay.cs

```diff
@@ -125,7 +125,7 @@ public void NavigateTo()
                 // Document tabs opened by NavigateTo are carefully created as preview or regular
                 // tabs by them; trying to specifically open them in a particular kind of tab here
                 // has no effect.
-                navigationService.TryNavigateToSpan(workspace, document.Id, _searchResult.NavigableItem.SourceSpan);
+                navigationService.TryNavigateToSpan(workspace, document.Id, _searchResult.NavigableItem.SourceSpan, reopenDocument: true);
```

Comment:
How does this interact with the checkbox in Tabs and Windows to enable or disable opening of Navigate To within preview windows?